backend/src/packages/users/user.entity.ts

```diff
@@ -112,14 +125,20 @@ class UserEntity implements Entity {
     fullName: string;
     passwordHash: string;
     passwordSalt: string;
+    isSurveyCompleted: boolean;
   } {
     return {
       email: this.email,
       fullName: this.fullName,
       passwordHash: this.passwordHash,
       passwordSalt: this.passwordSalt,
+      isSurveyCompleted: this.isSurveyCompleted,
     };
   }
+
+  public updateSurveyCompleted(isSurveyCompleted: boolean): void {
+    this.isSurveyCompleted = isSurveyCompleted;
+  }
```

Comment:
let's do this without additional methods
```suggestion
```

Reply:
@Liza-Veis Could, you, please clarify this? I did it in this way because I can't understand how can I update this without a method(I tried it like this: https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/66#discussion_r1306710894, but also get failed), because I need to pass the user entity to updateIsSurveyCompleted method in the user entity as you asked me before, so for that I need to have it with hashPassword and other unnecessary properties for me. 
So, I'm stuck on this a little bit.
I would appreciate it if you could explain this.

Reply:
```
  const item = await this.userRepository.updateIsSurveyCompleted(
      UserEntity.initialize({
        // other props
        isSurveyCompleted: true
      }),
    );
```

Reply:
Oh, I see, we won't have the whole user, then let's do it as it was before, just pass userId and value fo isSurveyCompleted